exercises/word-search/word_search_test.go

```diff
@@ -51,6 +51,7 @@ func TestSolve(t *testing.T) {
 	}
 	actual, err := Solve(words, puzzle)
 	if err != nil {
+		var _ error = err
 		t.Fatalf("Didn't expect error but got %v", err)
 	}
 	if !reflect.DeepEqual(actual, positions) {
```

Comment:
note for future:

In case any test cases start to actually expect error (I can't imagine what would), in that case we would want to only check this if `err == nil`